.travis.yml

```diff
@@ -14,7 +14,7 @@ matrix:
     - python: "3.5"
       env:
         - SPARK_VERSION=2.3.3
-        - PANDAS_VERSION=0.19.2
+        - PANDAS_VERSION=0.23.0
```

Comment:
Looks like we don't have to test against Pandas 0.19.2:

https://github.com/databricks/spark-pandas/blob/44ff66babdc30dabbdecbf6a140da9ff045d9e94/setup.py#L48